test/stubs.py

```diff
@@ -26,8 +26,8 @@
 from subscription_manager.cert_sorter import CertSorter
 from subscription_manager.cache import EntitlementStatusCache, ProductStatusCache, \
         OverrideStatusCache, ProfileManager, InstalledProductsManager, ReleaseStatusCache, \
-        PoolStatusCache, ContentAccessModeCache, SupportedResourcesCache, AvailableEntitlementsCache, \
-        SyspurposeValidFieldsCache, CurrentOwnerCache
+        PoolStatusCache, ContentAccessModeCache, SupportedResourcesCache, ReadThroughInMemoryCache, \
```

Comment:
I don't think we should still keep the ReadThroughInMemoryCache (despite how much I liked it) as it wasn't kept in master b/c we needed a different validity period for the ContentAccessModeCache (4 hours rather than once per run).

Reply:
That was one of the challenges with these cherry-picks - First was implementing ReadThroughInMemoryCache (and everything associated with it) and the next cherry-pick removed it.  I didn't want to remove it from the code until I had confirmation.